specification/common-types/resource-management/v5/types.json

```diff
@@ -488,7 +561,29 @@
       ],
       "x-ms-enum": {
         "name": "SkuTier",
-        "modelAsString": false
+        "modelAsString": false,
```

Comment:
>false [](http://example.com/codeflow?start=25&length=5)

I don't know if we need to fix this, but..
I'm a bit puzzled seeing this set of explicit values - I don't know that all resource providers actually follow this set, and we've got modelAsString as false, which means its non-extensible right?
I guess it had to be that way to be back compat with old definitions, since it was already modelAsString false, yet I am not sure what exactly we are imagining resource providers who don't follow this prescription will do in their definitions.